docs/changes.rst

```diff
@@ -0,0 +1,5 @@
+.. _pulp-2to3-migration-changes:
+
+.. include:: ../CHANGES.rst
+
+.. include:: ../HISTORY.rst
```

Comment:
I don't think we need a history file, since we haven't released anything yet (aka we don't have any history for release notes)

Reply:
I think it's ok to leave here since a) the user won't see a difference since the empty HISTORY.rst is included on this page and b) this is consistent with all the other sites. If you decided to remove it I'm ok w/ that too.